src/target/riscv/riscv.c

```diff
@@ -141,6 +141,10 @@ bool riscv_ebreaku = true;
 
 bool riscv_enable_virtual;
 
+bool enable_equality_match_trigger = true;
+bool enable_napot_trigger = true;
+bool enable_ge_lt_trigger = true;
+
```

Comment:
Ideally, these configuration flags should be per-target and not global -- that is, they should go into `struct riscv_info`. 

(The same applies to the already existing `riscv_ebreakX` bools above. They are global not on purpose, but because no one has moved them yet inside the riscv_info structure.)

Reply:
Addressed

Reply:
Thanks.